tests/components/roborock/mock_data.py

```diff
@@ -224,7 +224,599 @@
                     "desc": None,
                 },
             ],
-        }
+        },
+        {
```

Comment:
Not for this PR, but you could consider moving this out of code and store each of these devices in JSON fixtures instead.